app.py

```
@@ -0,0 +1,55 @@
+            summary_text = summarize_notes(notes)
+
+    return render_template(
+        "index.html",
+        notes=notes,
+        answer=answer_text,
+        links=links,
+        summary=summary_text   
+    )
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Ruff (0.15.7)

[error] Use of `debug=True` in Flask app detected

(S201)